flyteidl/src/lib.rs

```diff
@@ -0,0 +1,802 @@
+                .block_on(self.admin_service.list_node_executions(req))
+            {
+                Ok(res) => res,
+                Err(error) => panic!(
+                    "Failed at awaiting response from gRPC service server: {:?}",
+                    error
+                ),
+            })
+            .into_inner();
+            Ok(res)
+        }
+
+        pub fn get_node_execution_data(
+            &mut self,
+            req: crate::flyteidl::admin::NodeExecutionGetDataRequest,
+        ) -> PyResult<crate::flyteidl::admin::NodeExecutionGetDataResponse> {
+            let res = (match self
+                .runtime
+                .block_on(self.admin_service.get_node_execution_data(req))
+            {
+                Ok(res) => res,
+                Err(error) => panic!(
+                    "Failed at awaiting response from gRPC service server: {:?}",
+                    error
+                ),
+            })
+            .into_inner();
+            Ok(res)
+        }
+
+        pub fn list_task_executions(
+            &mut self,
+            req: crate::flyteidl::admin::TaskExecutionListRequest,
+        ) -> PyResult<crate::flyteidl::admin::TaskExecutionList> {
+            let res = (match self
+                .runtime
+                .block_on(self.admin_service.list_task_executions(req))
+            {
+                Ok(res) => res,
+                Err(error) => panic!(
+                    "Failed at awaiting response from gRPC service server: {:?}",
+                    error
+                ),
+            })
+            .into_inner();
+            Ok(res)
+        }
+
+        pub fn get_task_execution(
+            &mut self,
+            req: crate::flyteidl::admin::TaskExecutionGetRequest,
+        ) -> PyResult<crate::flyteidl::admin::TaskExecution> {
+            let res = (match self
+                .runtime
+                .block_on(self.admin_service.get_task_execution(req))
+            {
+                Ok(res) => res,
+                Err(error) => panic!(
+                    "Failed at awaiting response from gRPC service server: {:?}",
+                    error
+                ),
+            })
+            .into_inner();
+            Ok(res)
+        }
+
+        pub fn get_task_execution_data(
+            &mut self,
+            req: crate::flyteidl::admin::TaskExecutionGetDataRequest,
+        ) -> PyResult<crate::flyteidl::admin::TaskExecutionGetDataResponse> {
+            let res = (match self
+                .runtime
+                .block_on(self.admin_service.get_task_execution_data(req))
+            {
+                Ok(res) => res,
+                Err(error) => panic!(
+                    "Failed at awaiting response from gRPC service server: {:?}",
+                    error
+                ),
+            })
+            .into_inner();
+            Ok(res)
+        }
+    }
+}
+
+// #[macro_export]
+// macro_rules! concrete_generic_structure {
```

Comment:
At present, we don't need to specify the concrete type for generics parameterization, since we output the wrapped `FlyteRemote` as binding class instead of each gRPC stubs. So the macro `oncrete_generic_structure` is commented out.